k8s/upgrades/README.md

```diff
@@ -67,6 +80,14 @@ machine that has access to Kubernetes master**
 Upgrade steps vary depending on the way OpenEBS was installed by you.
 Below are steps to upgrade using some common ways to install OpenEBS:
 
+### Prerequisite for control plane upgrade
+1. Make sure all the blockdevices that are in use by cstor or localPV are connected to the node.
+2. Make sure all the blockdevices that are in use are in active and claimed state.
+3. Make sure that all manually created and claimed blockdevices are excluded in the NDM configmap path
+filter.
+
+**NOTE: Upgrade of LocalPV rawblock volumes are not supported. Please exclude it in configmap**
```

Comment:
If any of the above pre-requisites are not met? what should the user do? Reach out support via slack? 

Also, for each of the steps, is there a kubectl command that can be added that user can run and verify that pre-requisite is met? If the instructions are long, a new pre-req.md file can be created with detailed instructions. And a link added here.

Reply:
Done the changes.